### PR TITLE
Revert split action list item - Use title as link

### PR DESCRIPTION
**Pull request by tidy-dev**
Based on #13317 

## Description

The click list item to view online did not quite fit the Desktop UI/felt a bit unexpected. Switched back to the list item expand/collapsing and hover of title appears as link with blue text.

Got more UI feedback for using links as opposed to link external icon on private issue since team sync.

### Screenshots

https://user-images.githubusercontent.com/75402236/141828886-1c6d1c1e-fb99-4ed0-b04f-5111877c6429.mov

## Release notes
Notes: no-notes

### CODE REVIEW

File: app/src/ui/lib/tooltip.tsx [hunk 1]
```diff
@@ -7,6 +7,16 @@ import { assertNever } from '../../lib/fatal-error'
...
+export enum TooltipDirectionDef {
+  NORTH = 'n',
+  NORTH_EAST = 'ne',
+  EAST = 'e',
+  SOUTH_EAST = 'se',
+  SOUTH = 's',
+  SOUTH_WEST = 'sw',
+  WEST = 'w',
+  NORTH_WEST = 'nw',
+}
```
Comment (sergiou87) [inline app/src/ui/lib/tooltip.tsx]: Why do we need two of these? Can't we just use the `enum`?

Or, at least, maybe build the `TooltipDirection` type using the enum values, no? 🤔

**Review by sergiou87 — COMMENTED** [review]
Code LGTM! Only one minor thing.

I also tested it and it feels sooo much better than adding the icon that we discussed, I like it! 😍

File: app/src/ui/lib/tooltip.tsx (hunk 1 above)
Comment (tidy-dev) [reply app/src/ui/lib/tooltip.tsx]: .... Fear of regression. I just pushed a commit that updates 11 files to do this. :) I do prefer the enum everywhere.

**Review by sergiou87 — APPROVED** [review]
Thanks for those changes! They look good, and I tested it and didn't find regressions and worked great too 🐸 🕴️ ❗

**Pull request merged**